py3-cassandra-medusa.yaml

```diff
@@ -0,0 +1,67 @@
+# Generated from https://pypi.org/project/cassandra-medusa/
+package:
+  name: py3-cassandra-medusa
+  version: 0.17.1
+  epoch: 0
+  description: Apache Cassandra backup and restore tool
+  copyright:
+    - license: Apache
```

Comment:
Looks like the license name was not created correctly here, can we check it and complete?

Reply:
Yes, its Apache License 2.0, I will update this, thanks Mark